packages/server/graphql/types/EstimateStage.ts

```diff
@@ -130,7 +132,13 @@ const EstimateStage = new GraphQLObjectType<Source, GQLContext>({
             info,
             provider.serverBaseUrl!
           )
-          const [issueData] = await manager.getIssue({gid})
+          const [issueData, issueError] = await manager.getIssue({gid})
+          if (issueError) {
```

Comment:
This fixes the error mentioned in https://github.com/ParabolInc/parabol/issues/6511 where a user can't join a meeting. 

If there are many issues selected, GitLab will stop returning `issueData` and return `issueError` saying that the query complexity has exceeded the limit of 250. By returning a null field here, we prevent the app from crashing.

However, if you select all issues in GitLab and refresh the page, GitLab can't retrieve all the issues and shows the dummy error text in the sidebar. I'm working on fixing this in https://github.com/ParabolInc/parabol/pull/6520